.gitignore

```diff
@@ -22,3 +22,4 @@ sdk/src/main/res/values/developer-config.xml
 node_modules/
 gh_token.txt
 android-docs-repo
+/scripts/revapi/revapi-0.9.7/
```

Comment:
Is this a temp folder? What if the version of the revapi is changed?

Reply:
This is not needed anymore, will delete it.